package-lock.json

```diff
@@ -27228,12 +27228,6 @@
         "safe-buffer": "^5.1.1"
       }
     },
-    "read-file": {
-      "version": "0.2.0",
-      "resolved": "https://registry.npmjs.org/read-file/-/read-file-0.2.0.tgz",
-      "integrity": "sha1-cMa6+IQux9FUD5gf0Oau1Mgb1UU=",
-      "dev": true
-    },
```

Comment:
Unintentional?

Reply:
Seems like someone else left this out of their PR. I've removed it from this PR, but it will eventually need to go in.